mxcube3/ui/actions/general.js

```diff
@@ -18,22 +18,48 @@ export function showErrorPanel(show, message = '') {
 
 export function getInitialStatus() {
   return function (dispatch) {
-    fetch('mxcube/api/v0.1/initialstatus', {
+    let state = {};
+
+    let motors = fetch('mxcube/api/v0.1/diffractometer/movables/state', {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        'Accept': 'application/json',
+        'Content-type': 'application/json'
+      }
+    });
+    let beamInfo = fetch('mxcube/api/v0.1/beam/info', {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        'Accept': 'application/json',
+        'Content-type': 'application/json'
+      }
+    });
+    let sampleVideoInfo = fetch('mxcube/api/v0.1/sampleview/camera/info', {
       method: 'GET',
       credentials: 'include',
       headers: {
         'Accept': 'application/json',
         'Content-type': 'application/json'
       }
-    }).then(function (response) {
-      if (response.status >= 400) {
-        throw new Error('Server refused to send initialstatus');
+    });
+    let diffractometerInfo = fetch('mxcube/api/v0.1/diffractometer/info', {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        'Accept': 'application/json',
+        'Content-type': 'application/json'
       }
-      return response.json();
-    })
-    .then(function (json) {
-      dispatch(setInitialStatus(json));
     });
 
+    motors.then(response => { state.Motors = response.json() });
```

Comment:
Thanks for this - I didn't know. One more thing I find weird in this fetch api !
I am going to fix it asap.